requirements.in

```diff
@@ -8,4 +8,5 @@ psycopg2==2.8
 prometheus_client==0.7
 django-environ
 drf-spectacular>=0.27,<0.28
-djangorestframework-dataclasses>=1.3.1,<2
\ No newline at end of file
+djangorestframework-dataclasses>=1.3.1,<2
+django-debug-toolbar>=4.3,<4.4 # When upgrading DDT, check that newer versions still support Django 3.2.
```

Comment:
can this go in requirements_dev.in ?

Reply:
The issue with putting it in dev requirements, is in `urls.py` i'm doing:

```python
if settings.DEBUG:
    urlpatterns += [path("__debug__/", include("debug_toolbar.urls"))]
```

So, if we ever tried to use DEBUG=True without `dev_requirements.txt` being installed, it would presumably crash with a not-so-helpful error about missing imports or something.

I guess we could a test to see if the package is installed, e.g.:
```
try:
    import debug_toolbar
    debug_toolbar_available = True
except ImportError:
    debug_toolbar_available = False

if debug_toolbar_available and settings.DEBUG:
    urlpatterns += [path("__debug__/", include("debug_toolbar.urls"))]
```

What do you think?